riptable/hypothesis_tests/test_categorical_property.py

```diff
@@ -71,27 +71,34 @@ def one_of_categorical_values(draw):
     return draw(one_of(cat_values))
 
 
-@pytest.mark.skipif(
-    is_running_in_teamcity(), reason="Categorical generator needs to be rewritten for better performance before re-enabling this test to run in TeamCity builds."
-)
+@pytest.mark.skipif(reason="Categorical generator needs to be rewritten for better performance before re-enabling this test to run in TeamCity builds.")

```

Comment:
should be `skip` instead of `skipif`, but it will skip since reason is not an empty string